native/common/include/jp_buffertype.h

```diff
@@ -0,0 +1,30 @@
+#ifndef _JPBUFFERTYPE_H_
+#define _JPBUFFERTYPE_H_
+
+/**
+ * Class to wrap Java Class and provide low-level behavior
+ */
+class JPBufferType : public JPClass
+{
+public:
+	JPBufferType(JPJavaFrame& frame, jclass cls, const string& name, JPClass* superClass, const JPClassList& interfaces, jint modifiers);
```

Comment:
Actually it was fine to have one argument per line, I just thought it was not aligned with the first bracket. Sorry for wasting your time here.

Reply:
Netbeans has some strange behavior when it comes to mixed spaces and tabs resulting in odd results.   I try when a list of arguments gets too long to follow to split them out, but it only lines up properly if the editor is set with the same tab width that I am using and even then sometimes it decides to randomly respace it after typesetting.   I know that netbeans is not a real C++ editor, but when diagnosing code in 3 languages at a time it does serve a purpose.  Like the "}  ;"  I just have to accept the good stuff it is doing like cross referencing between the three open projects and ignore the weird.